src/items.cpp

```diff
@@ -66,6 +66,8 @@ const std::unordered_map<std::string, ItemParseAttributes_t> ItemParseAttributes
 	{"transformequipto", ITEM_PARSE_TRANSFORMEQUIPTO},
 	{"transformdeequipto", ITEM_PARSE_TRANSFORMDEEQUIPTO},
 	{"duration", ITEM_PARSE_DURATION},
+	{"durationmin", ITEM_PARSE_DURATION},
```

Comment:
Shouldn't this be ITEM_PARSE_DURATION_MIN?

Reply:
yes

Reply:
What happens is that I want to reuse the same variable, for this case, since `duration` and `durationmin` are the same, only to make it look more beautiful in the xml file, add a derivative that can be called `durationmin` so that it matches `min` <-> `max`

Reply:
oh true, I forgot that they are the same, disregard my previous comment then